docs/reference/glossary.md

```diff
@@ -12,7 +12,10 @@ DKMS
   See the dkms manpages for more information.
 
 ABI
-  Application binary interface.
+  Application binary interface, or ABI defines a stable interface between user-space
```

Comment:
LGTM.  Just one nit..
```suggestion
  Application Binary Interface, or ABI defines a stable interface between user-space
```

Reply:
Thank you very much for the feedback. As  a beginner, I'm  learning a lot about kernel documentation and open source contribution in general.

I have added the requested changes. Let me know for any improvements I should work on.

Reply:
Thanks @Jetoson , we're so glad to have you join us at the start of your open source journey.
Please continue to contribute if something interests you in our Kernel docs.

On a related note, Canonical has the Open Documentation Academy, which is a platform for you to connect with the Documentation team at Canonical, and also a place to start getting involved in open source through documentation.
Read more about the ODA here:
https://canonical.com/documentation/open-documentation-academy

There are also specially curated issues which anyone who is interested can contribute to here, as part of the ODA efforts:
https://github.com/canonical/open-documentation-academy/issues

Let me know if you have any questions.

Thanks!

- Anne -